style.css

```diff
@@ -226,16 +226,24 @@ label[for='toggle-all'] {
 	transition: color 0.2s ease-out;
 }
 
-.todo-list li .destroy:hover {
-	color: #af5b5e;
+@media (any-pointer: fine) {
+  .todo-list li .destroy:hover {
+    color: #af5b5e;
+  }
+  .todo-list li:hover .destroy {
+    display: block;
+  }
 }
 
-.todo-list li .destroy:after {
-	content: '×';
+@media (any-pointer: coarse) {
+  .todo-list li .destroy {
+    color: #af5b5e;
+    display: block;
+  }
 }
 
-.todo-list li:hover .destroy {
-	display: block;
+.todo-list li .destroy:after {
+	content: '×';
```

Comment:
Are these changes the result of updating `style.css` from what was embedded?

Reply:
Yes, the only other place `style.css` is used is in [bench.hs](https://github.com/reflex-frp/reflex-todomvc/blob/cd15a37b0e6decf42840967ce5fba6a03cf278fa/src-bin/bench.hs), and I trust these few changed bytes won't affect the benchmark

Reply:
No point on trying to fudge the benchmark by faking out your css anyway :stuck_out_tongue:

Reply:
Hopefully https://github.com/reflex-frp/reflex-todomvc/pull/16 didn't change it!

Reply:
The latest commit is a direct copy of the latest embedded string :)